container-metrics.html.md.erb

```diff
@@ -104,13 +104,17 @@ memory usage:   256M
 #0   running   2019-04-05T17:41:31Z   0.4%   39.5M of 256M   89.9M of 1G
 </pre>
 
-### <a id="cf-app"></a> Retrieve CPU Entitlement Metrics (Experimental)
+Using the `cf app APP_NAME` command for your application will show you what percentage of CPU the app is currently using relative to the total CPU on the host machine.
+
+### <a id="cf-app"></a> Retrieve CPU Entitlement Metrics
+
+The new `AbsoluteCPUEntitlement` metric shows the amount of CPU usage relative to an app's entitlement. 
```

Comment:
I don't like using the word "new" because it doesn't remain true for long.

```suggestion
The metric `AbsoluteCPUEntitlement` shows an app's CPU usage relative to an its entitlement.
```